### ISSUE TITLE
v-show directives requiring that attribute value

### ISSUE THREAD

**Issue by Wanjiru-faith** [vuejs/eslint-plugin-vue#902]
**Tell us about your environment**	
* **ESLint version:**	
* **eslint-plugin-vue version:**	
* **Node version:**	

**Please show your full configuration:**

```json

```

**What did you do?**	

```vue

```

**What did you expect to happen?**

**What actually happened?**

**Comment by Wanjiru-faith** [comment]
'v-show' directives require that attribute value (vue/valid-v-show)
